hardware_interface/test/interface_manager_test.cpp

```diff
@@ -84,6 +84,93 @@ TEST(InterfaceManagerTest, InterfaceRewriting)
   EXPECT_EQ(2, foo_iface_ptr->foo);
 }
 
+struct TestCombinationHandle {
+  TestCombinationHandle(const std::string& name) : name{name} {}
+
+  std::string getName() const
+  {
+    return name;
+  }
+
+  std::string name;
+};
+
+struct TestCombinationInterface : public ResourceManager<TestCombinationHandle>
+{
+
+};
```

Comment:
Let's make this a class.

```suggestion
class TestCombinationInterface : public ResourceManager<TestCombinationHandle> {};
```

Reply:
Applied in rebased commits